Formula/p/pact-cli.rb

```diff
@@ -0,0 +1,32 @@
+class PactCli < Formula
+  desc "CLI client for the Pact Broker"
+  homepage "https://github.com/pact-foundation/pact_broker-client"
+  url "https://github.com/pact-foundation/pact_broker-client/archive/refs/tags/v1.77.0.tar.gz"
+  sha256 "a7f333afee6eae511c177256a284fc30d7b236bc4ab715107303d99353555664"
+  license "MIT"
+
+  depends_on "ruby"
+
+  def install
+    ENV["BUNDLE_VERSION"] = "system"
+    ENV["GEM_HOME"] = libexec
+
+    inreplace "pact-broker-client.gemspec",
+              /`git ls-files`.split.*$/,
+              'Dir["lib/**/*", "bin/*", "README*", "LICENSE*"]'
```

Comment:
this removes files the maintainers and users expect to be there  (examples/doc folders)

it would be far better to propose this in the source repo itself if required